File: cpp_ros_topic/src/ros_topic/src/publisher_node.cpp
#include "ros/ros.h"
#include "ros_topic/person_data.h"

int main(int argc, char **argv){
    ros::init(argc, argv, "Publisher");
    ros::NodeHandle nh;
    
    ros::Publisher topic_pub = nh.advertise <ros_topic::person_data>("ros_topic_latihan", 1000);
    ros::Rate loop_rate(1);

    while(ros::ok()){
        ros_topic::person_data person_data;
        person_data.usia = 18;
        person_data.tinggi = 174.5;
        person_data.nama = "Sabilul Huda";
        person_data.kehadiran = true;

        topic_pub.publish(person_data);
        ros::spinOnce();
        loop_rate.sleep();
    };

    return 0;
}

File: python_ros_topic/src/tutorials/scripts/publisher_node.py
#!/opt/ros/noetic/bin
import rospy
from std_msgs.msg import String
from tutorials.msg import Position

def talk_to_me():
    pub = rospy.Publisher("talking_topic", Position, queue_size = 10)
    rospy.init_node('publisher_node', anonymous = True)
    rate = rospy.Rate(1)
    rospy.loginfo("Publisher node started, now publishing messages")
    while not rospy.is_shutdown():
        msg = Position()
        msg.usia = 18
        msg.tinggi = 174.5
        msg.nama = "Sabilul Huda"
        msg.kehadiran = True
        pub.publish(msg)
        rate.sleep()

if __name__ == '__main__':
    try:
        talk_to_me()
    except rospy.ROSInterruptException:
        pass